### PR TITLE
WIP: Updated lab to release 2.0

### PR DESCRIPTION
**Pull request by mvazquezc**

### CODE REVIEW

**Comment by cdoan1** [comment]
@mvazquezc thanks for doing this! I also ran into this

```
diff --git a/multicloud-deployment/acm/02_aws_placement_rule.yaml b/multicloud-deployment/acm/02_aws_placement_rule.yaml
index 669aa04..5d26f5a 100644
--- a/multicloud-deployment/acm/02_aws_placement_rule.yaml
+++ b/multicloud-deployment/acm/02_aws_placement_rule.yaml
@@ -5,7 +5,8 @@ metadata:
   namespace: apache-test
 spec:
   clusterConditions:
-   - type: OK
+    - status: 'True'
+      type: ManagedClusterConditionAvailable
   clusterSelector:
     matchExpressions: []
     matchLabels:
```

**Comment by mvazquezc** [comment]
> @mvazquezc thanks for doing this! I also ran into this
> 
> ```
> diff --git a/multicloud-deployment/acm/02_aws_placement_rule.yaml b/multicloud-deployment/acm/02_aws_placement_rule.yaml
> index 669aa04..5d26f5a 100644
> --- a/multicloud-deployment/acm/02_aws_placement_rule.yaml
> +++ b/multicloud-deployment/acm/02_aws_placement_rule.yaml
> @@ -5,7 +5,8 @@ metadata:
>    namespace: apache-test
>  spec:
>    clusterConditions:
> -   - type: OK
> +    - status: 'True'
> +      type: ManagedClusterConditionAvailable
>    clusterSelector:
>      matchExpressions: []
>      matchLabels:
> ```

Hey Chris, I had to change all subscriptions to the new clusterConditions format. You can find the updates files in this other PR: 

https://github.com/RHsyseng/acm-app-lifecycle-policies-lab/pull/1/files

File: introduction-to-gitops-and-policies/04_using_timewindows.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ Now we should have our application running on the production cluster:
 
 ~~~sh
 date
-oc --context spoke2 -n gitops-apps get pods,svc,route
+oc --context managed-prod -n gitops-apps get pods,svc,route
 ~~~
```
[the remark this answers is not in the document]
Comment (mvazquezc) [reply introduction-to-gitops-and-policies/04_using_timewindows.md:73]: Added a note.

File: introduction-to-gitops-and-policies/05_disaster_recovery.md [hunk 2]
```diff
@@ -5,27 +5,22 @@ In this use case we are going to define the deployment of our application as fol
...
-    ~~~sh
-    oc --context hub delete -f https://github.com/RHsyseng/acm-app-lifecycle-policies-lab/raw/master/acm-manifests/reversewords-kustomize/08_subscription-timewindow.yaml
-    ~~~ -->
 1. To avoid app creation collisions we are going to delete previous subscriptions and applications
 
     ~~~sh
-    oc --context hub delete -f https://github.com/RHsyseng/acm-app-lifecycle-policies-lab/raw/master/acm-manifests/reversewords-kustomize/07_subscription-all-okay.yaml
+    oc --context hub delete -f https://github.com/RHsyseng/acm-app-lifecycle-policies-lab/raw/master/acm-manifests/reversewords-kustomize/08_subscription-timewindow.yaml
     ~~~
 2. Label the clusters
```
[the remark this answers is not in the document]
Comment (mvazquezc) [reply introduction-to-gitops-and-policies/05_disaster_recovery.md:17]: Added

File: introduction-to-gitops-and-policies/05_disaster_recovery.md [hunk 3]
```diff
@@ -36,23 +31,25 @@ since we defined `clusterReplicas: 1` within the `PlacementRule`.
...
+    ~~~
+    > ![WARNING](assets/warning-icon.png) **NOTE:** The application will be deployed to `managed-cluster1-dev` cluster only based on the output above
 5. Create the `Subscription`
 
     ~~~sh
     oc --context hub create -f https://github.com/RHsyseng/acm-app-lifecycle-policies-lab/raw/master/acm-manifests/reversewords-kustomize/10_subscription-finance.yaml
     ~~~
 
-Now we should have our application running on the `spoke` cluster and not in `spoke2` cluster:
+Now we should have our application running on the `managed-cluster1-dev` cluster and not in `managed-cluster2-prod` cluster:
```
[the remark this answers is not in the document]
Comment (mvazquezc) [reply introduction-to-gitops-and-policies/05_disaster_recovery.md:46]: Modified the wording here to point that the app will be present in the cluster decided by the placementrule.

**Pull request merged**